README.md

```diff
@@ -69,8 +77,8 @@ We encourage you to take a look `at test/index.js` to see how the functions abov
 ```
 git clone https://github.com/emscripten-core/emsdk.git
 cd emsdk
-./emsdk install sdk-1.38.8-64bit
-./emsdk activate sdk-1.38.8-64bit
+./emsdk install 1.39.19
```

Comment:
by updating emscripten, the fix to remove unhandled rejection handler is probably redundant and can be replaced by a dedicated flag NODEJS_CATCH_REJECTION which was introduced in later versions: https://github.com/vacuumlabs/cardano-crypto.js/blob/ae8cdfac8d33df5ffe13a813412af83fb71d4225/build.js#L37

The flag was originally added because a user of our library reported issues related to catching unhlandled rejections: https://github.com/emscripten-core/emscripten/issues/7855